main.go

```diff
@@ -64,20 +63,19 @@ func New() *RootModule {
 
 // NewModuleInstance implements the modules.IsModuleV2 interface to return
 // a new instance for each VU.
-func (*RootModule) NewModuleInstance(m k6modules.InstanceCore) k6modules.Instance {
+func (*RootModule) NewModuleInstance(vu k6modules.VU) k6modules.Instance {
 	return &ModuleInstance{
-		InstanceCore: m,
 		mod: &JSModule{
-			InstanceCore: m,
-			Devices:      common.GetDevices(),
-			Version:      version,
+			vu:      vu,
+			Devices: common.GetDevices(),
+			Version: version,
 		},
 	}
 }
 
 // GetExports returns the exports of the JS module so that it can be used in
```

Comment:
The comment `GetExports` -> `Exports`

Reply:
:facepalm: Thanks, we should really prioritize #58.